src/main/java/frc/robot/subsystems/swerve/GyroIO.java

```diff
@@ -0,0 +1,29 @@
+// Copyright 2023-2025 FRC 8033
+//
+// This program is free software; you can redistribute it and/or
+// modify it under the terms of the GNU General Public License
+// version 3 as published by the Free Software Foundation or
+// available in the root directory of this project.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+package frc.robot.subsystems.swerve;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import org.littletonrobotics.junction.AutoLog;
+
+public interface GyroIO {
+  @AutoLog
+  public static class GyroIOInputs {
+    public boolean connected = false;
```

Comment:
```suggestion
    public boolean isConnected = false;
```

Suggestion. Would require refactor of everything else. It's a personal preference and up to you since we don't set code base Boolean standards